orm/solid-start/lib/prisma.ts

```diff
@@ -0,0 +1,6 @@
+import { PrismaClient } from '@prisma/client'
+import { withAccelerate } from '@prisma/extension-accelerate'
+
+const prisma = new PrismaClient().$extends(withAccelerate()) // If you're not using Prisma Postgres, remove `.$extends(withAccelerate())`
```

Comment:
We can remove the comment as it's in our readme.
```suggestion
const prisma = new PrismaClient().$extends(withAccelerate())
```